api/src/middleware/mongoose.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const { fillErrorObject } = require('../middleware/error');
+
+async function validateObjectId(_id, type, res, next) {
+  if (!mongoose.Types.ObjectId.isValid(_id)) {
+    console.log('in here');
+    console.log(_id);
+    next(
+      fillErrorObject(400, 'Validation error has occurred', [
```

Comment:
It's a nit pick, but I think convention is to just use 'Validation error', i.e. no need for a proper sentence. But it's a really small change so don't think it would be worth going through all these and changing them?

Reply:
You're right, validation error does sound better. No worries, I'll go change it, that's what replace all is for. :)